docs/getting-started/faqs/mia-platform-microfrontend-composer-faqs.md

```diff
@@ -0,0 +1,185 @@
+sidebar_label: Microfrontend Composer FAQs
+---
+
+### Core Concepts & Architecture
+
+#### What is the Microfrontend Composer?
+The Microfrontend Composer is a low-code/no-code tool within the Mia-Platform Console that allows you to build user interfaces, such as backoffices, dashboards, and **developer portals**, by assembling pages from a library of pre-built or custom components. It provides a visual interface for designing layouts and configuring components, abstracting away much of the complexity of frontend development.
+[Discover more](/microfrontend-composer/what-is.md)
+
+#### What is the underlying technology that powers the Microfrontend Composer?
+The Composer uses **micro-lc**, an open-source microfrontend orchestrator, to render the final application. micro-lc is responsible for dynamically loading and displaying different types of content, including pages composed of web components, iframes, or entire microfrontends built with frameworks like React or Angular.
+[Discover more](/microfrontend-composer/architecture.md)
+
+#### Why should I use a microfrontend architecture?
+A microfrontend architecture breaks down a large monolithic frontend application into smaller, independent, and deployable pieces. The key benefits include:
+* **Team Autonomy**: Different teams can work on different parts of the application independently.
+* **Technology Agnosticism**: You can use different frameworks (React, Angular, Vue) for different microfrontends.
+* **Incremental Upgrades**: You can update or rewrite parts of your application without affecting the whole.
+* **Resilience**: An error in one microfrontend is less likely to bring down the entire application.
+  This approach aligns well with a modern **service oriented architecture**.
+  [Discover more](/microfrontend-composer/what-is.md)
+
+#### What is a Web Component?
+A Web Component is a standardized way to create custom, reusable HTML tags. They are framework-agnostic, meaning a web component can be used in any web application, regardless of whether it's built with React, Angular, or plain JavaScript. The Microfrontend Composer heavily relies on web components for its composable pages.
+[Discover more](/microfrontend-composer/external-components/overview.md)
+
+#### What is Back-Kit?
+**Back-Kit** is Mia-Platform's official library of pre-built, production-ready web components designed specifically for the Microfrontend Composer. It includes a wide range of components for building data-intensive applications, such as tables, forms, buttons, cards, and data clients. These components are designed to work together seamlessly through an event-based communication system.
+[Discover more](/microfrontend-composer/back-kit/10_overview.md)
+
+#### What is the difference between a "compose" page and a "microfrontend" page?
+* A **Compose Page** is built visually within the Composer by dragging, dropping, and configuring web components (like those from Back-Kit). This is the low-code approach.
```

Comment:
Since drag&drop functionalities are not yet part of the Composer, I would leave them out